src/apps/explorer/components/common/ExplorerTabs/ExplorerTabs.stories.tsx

```diff
@@ -47,7 +48,35 @@ const tabItems = [
   },
 ]
 
+const Wrapper = styled.div`
+  width: 500px;
+  padding: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px dashed darkorange;
+
+  .tab-extra-content {
```

Comment:
@alongoni Maybe you can help me [make this work](https://github.com/gnosis/gp-ui/blob/styling-tab-1043/src/components/common/Tabs/Tabs.tsx#L79).